.github/workflows/release.yml

```
@@ -141,9 +192,94 @@ jobs:
+      - name: Checkout Code
+        uses: actions/checkout@v6
+
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: '3.14'
+
+      - name: Generate Devcontainer Link
+        id: devcontainer_link
+        if: needs.publish-devcontainer.result == 'success'
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:10:6: Double quote to prevent globbing and word splitting

(shellcheck)